pwiz_tools/Skyline/TestUtil/PythonTestUtil.cs

```diff
@@ -185,15 +204,10 @@ public bool InstallPython(BuildLibraryDlg buildLibraryDlg)
             return false;
         }
 
-        protected override void DoTest()
-        {
-            throw new NotImplementedException();
-        }
-
         private void RunNvidiaDialog(MultiButtonMsgDlg nvidiaDlg)
         {
             nvidiaDlg.ClickNo();
-            MessageDlg confirmDlg = WaitForOpenForm<MessageDlg>(600000);
+            MessageDlg confirmDlg = AbstractFunctionalTest.WaitForOpenForm<MessageDlg>(600000);
             Assert.AreEqual(string.Format(ToolsUIResources.PythonInstaller_OkDialog_Successfully_set_up_Python_virtual_environment),
                 confirmDlg.Message);
             confirmDlg.OkDialog();
```

Comment:
Again, you should use the change I provided privately:
`            AbstractFunctionalTest.OkDialog(confirmDlg, confirmDlg.OkDialog);
`
this achieves two things - runs UI code on the UI thread, and makes sure the dialog is completely dismissed before moving along.